gems/smithy-client/lib/smithy-client/input.rb

```diff
@@ -65,7 +65,7 @@ def initialize(options = {})
       #   a request that may return a large payload that you don't want to
       #   load into memory.
       #
-      # @return [Output]
+      # @return [Output, nil]
```

Comment:
What cases does this return nil? Shouldn't there always be an output (except ofc when an error is raised, but thats a different control flow)?

Reply:
If there are no handlers, it can be nil. In practice this doesn't happen but satisfies linting.